### PR TITLE
Swagger UI

### PR DESCRIPTION
**Pull request by lebe1**
# Type of change

- [x] New feature (non-breaking change which adds functionality)

# Description

This PR adds a Swagger to the backend.
It is available via `localhost:3000/swagger/`
New dependency is [swaggo](https://github.com/swaggo/swag) now

# Checklist:

- [x] I have commented my code (or ChatGPT did), particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings, neither in my IDE nor in my browser
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by lebe1** [comment]
@nanu-c I will fix that testing error in my next PR when I also change both API calls receiving their data from database

File: app/docs/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    email: support@swagger.io
+    name: API Support
+    url: http://www.swagger.io/support
+  description: This is a sample server celler server.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  termsOfService: http://swagger.io/terms/
+  title: Swagger Example API
+  version: 0.0.1
```
Comment (nanu-c) [inline app/docs/swagger.yaml:36]: same here

File: app/handlers/handlers.go [hunk 2]
```diff
@@ -1,13 +1,45 @@
...
+//	@license.url	http://www.apache.org/licenses/LICENSE-2.0.html
+
+//	@host		localhost:3000
+//	@BasePath	/api/
+
+//	@securityDefinitions.basic	BasicAuth
+
+//	@externalDocs.description	OpenAPI
+//	@externalDocs.url			https://swagger.io/resources/open-api/
+
```
Comment (nanu-c) [inline app/handlers/handlers.go:20]: Okey I understand the two other files are generated out of this.

**Review by nanu-c — CHANGES_REQUESTED** [review]
Everything else looks good!

**Pull request merged**